browser/src/main.js

```diff
@@ -0,0 +1,100 @@
+class PMA {
+  /**
+   * @param {object} configuration - An object
+   * @param {string} configuration.brandId - An string with the client brand id
+   * @param {string} configuration.redirectTo - url client app, it will be used to redirect at the end of the flow
+   * @param {string} configuration._domain - change iframe and location domain only for testing purposes
+   */
+  constructor({ brandId, redirectTo = window.location, _domain = process.env.DOMAIN }) {
```

Comment:
do we want the `redirectTo` to default rather than fail ?

Reply:
also `_domain` or are they optional ?

Reply:
`redirectTo` is mandatory.
`_domain` is optional but also is a private property and you can change it at your own risk